src/Microsoft.AspNet.Mvc.Core/IUrlHelper.cs

```diff
@@ -33,10 +33,11 @@ public interface IUrlHelper
         string Content(string contentPath);
 
         /// <summary>
-        /// Returns a value that indicates whether the URL is local.
+        /// Returns a value that indicates whether the URL is local. An absoulte URL is considered local
```

Comment:
If I'm not mistaken, an `absolute URL` **always** has a host/authority section. What we're talking about is a URL with an absolute *path*.